qutip/core/coefficient.py

```diff
@@ -191,6 +191,18 @@ def conj(coeff):
     return ConjCoefficient(coeff)
 
 
+def const(value):
+    """ return a Coefficient with a constant value.
+    """
+    return ConstantCoefficient(value)
+
+
+# XXX: HACKITY HACK
```

Comment:
@Ericgig Is this a nice way to expose these extra coefficient constructors at the top level of QuTiP, or too hacky? If you like it, I can add tests.

Reply:
I have more a question about why they should be exposed.
Coefficient are seldom manipulated directly and the name of these functions is too common to make available.
So having them available only when manually imported seems good enough.
The method `Coefficient.conj()` is giving the same effect and is already available.

Reply:
I agree that `conj` is already easily available. `norm` seems fairly useful and at the moment it requires knowing to import it from `qutip.core.coefficient`. `const` isn't that useful for `QobjEvo` but is very useful if one has to work with coefficients more directly.

Since this seems a little bit contentious, let me just revert this part for now and import `const` manually where I need it for `nm_mcsolve`.